util/scripts/build-container-image.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+basedir=$(dirname "$0")
+cd "$basedir"/../..
+
+NODE_VERSION=19.3
+RUST_VERSION=1.66
+
+CONTAINER_PLATFORM="${CONTAINER_PLATFORM:-linux/amd64}"
+RUST_TARGET="${RUST_TARGET:-x86_64-unknown-linux-musl}"
+
+BUILT_TIME_UTC="$(date -u +"%Y-%m-%dT%TZ")"
+GIT_COMMIT_HASH="$(git rev-parse --short HEAD || echo "unknown")"
+VERSION="$(grep -e '^version' backend/Cargo.toml | cut -d '=' -f 2- | tr -d ' "')"
```

Comment:
FWIW this is used in `make-release.sh`:

```sh
sed -n -E 's/^version = "([^"]+)"$/\1/p' backend/Cargo.toml
```

Dunno if either one of those is "better". One could also extract this into `print-version.sh` or sth, but it's also not very important :D